config.js

```diff
@@ -45,8 +45,9 @@ function getConfig(env) {
             helperAccount: 'guildnet',
         };
     case 'local':
+    case 'localnet':
         return {
-            networkId: 'local',
+            networkId: 'localnet',
```

Comment:
This will be a breaking change, I would rather avoid it. Do you have a strong motivation to merge it? Can you describe a use case?

Reply:
I tested this extensively and it indeed caused issues in other components. I rolled back this change.

Reply:
Thanks!